### PR TITLE
feat: created profile page

### PR DESCRIPTION
**Pull request by Rockyraven**
feat: created profile page
some css update
some material icon added

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=arjunkumar
```
Comment (arjun-computer-geek) [inline .env:1]: add this to gitignore file, because this is a secret thing.

File: src/component/navbar/navbar.css [hunk 2]
```diff
@@ -14,20 +15,32 @@
...
     height: 3rem;
+    background-color: var(--clr-page-bg);
 }
 .search-button{
     padding: 0.9rem;
+    background-color: var(--clr-page-bg);
+
+}
+.login{
+    color: var(--icon-color);
```
Comment (arjun-computer-geek) [inline src/component/navbar/navbar.css:42]: nicely use css variables.

File: src/component/sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -10,45 +10,45 @@ import ThumbUpOffAltOutlinedIcon from "@mui/icons-material/ThumbUpOffAltOutlined
 
 export const Sidebar = () => {
   return (
-    <div className="sidebar">
+    
       <div className="sidebar-container">
         <Link to="/home">
-          <div>
+          <div className="link" >
```
Comment (arjun-computer-geek) [inline src/component/sidebar/Sidebar.jsx:16]: it's className can be better.

**Review by arjun-computer-geek — APPROVED** [review]
All are good. Good folder structure. Good job. @Rockyraven

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to me , you can proceed further

**Pull request merged**